app/src/main/java/com/kickstarter/ui/extensions/ImageViewExt.kt

```diff
@@ -3,75 +3,162 @@ package com.kickstarter.ui.extensions
+            FirebaseCrashlytics.getInstance().recordException(e)
         }
     }
 }
 
 fun ImageView.loadImage(url: String?) {
     url?.let {
-        Picasso
-            .get()
-            .load(it)
-            .into(this)
+        try {
+            Glide.with(context)
+                .load(url)
+                .placeholder(ColorDrawable(Color.TRANSPARENT))
+                .diskCacheStrategy(DiskCacheStrategy.RESOURCE)
+                .into(this)
+        } catch (e: Exception) {
+            this.setImageResource(R.drawable.image_placeholder)
+            FirebaseCrashlytics.getInstance().setCustomKey("ImageView.loadImage", " with url: $it ${e.message ?: ""}")
+            FirebaseCrashlytics.getInstance().recordException(e)
+        }
     }
 }
 
+fun ImageView.loadImageWithResize(
+    url: String?,
+    targetImageWidth: Int,
+    targetImageHeight: Int,
+    placeholder: Drawable
+) {
+    url?.let {
+        try {
+            Glide.with(context)
+                .load(url)
+                .diskCacheStrategy(DiskCacheStrategy.RESOURCE)
+                .apply(RequestOptions().override(targetImageWidth, targetImageHeight))
+                .centerCrop()
+                .placeholder(placeholder)
+                .into(this)
+        } catch (e: Exception) {
+            this.setImageResource(R.drawable.image_placeholder)
+            FirebaseCrashlytics.getInstance().setCustomKey("ImageView.loadImageWithResize", " with url: $it ${e.message ?: ""}")
+            FirebaseCrashlytics.getInstance().recordException(e)
+        }
+    }
+}
 fun ImageView.loadImage(url: String?, context: Context, imageViewPlaceholder: AppCompatImageView? = null) {
-    val target = this
-    if (context.applicationContext.isKSApplication()) {
-        Picasso
-            .get()
-            .load(url)
-            .into(
-                this,
-                object : Callback {
-                    override fun onSuccess() {
-                        imageViewPlaceholder?.setImageDrawable(target.drawable)
-                    }
+    url?.let {
+        val targetView = this
+        if (context.applicationContext.isKSApplication()) {
+            try {
+//                Picasso
+//                    .get()
+//                    .load(url)
+//                    .into(
+//                        this,
+//                        object : Callback {
+//                            override fun onSuccess() {
+//                                imageViewPlaceholder?.setImageDrawable(target.drawable)
+//                            }
+//
+//                            override fun onError(e: Exception?) {
+//                                target.setImageDrawable(null)
+//                                imageViewPlaceholder?.setImageDrawable(null)
+//                            }
+//                        }
+//                    )
```

Comment:
Did you mean to leave these comments in?

Reply:
nop!